install/configuring_inventory_file.adoc

```diff
@@ -1190,11 +1190,22 @@ the Docker daemon.
 and the Docker daemon.
 
 |`openshift_no_proxy`
-|This variable is used to set the `NO_PROXY` environment variable for masters
+a|This variable is used to set the `NO_PROXY` environment variable for masters
 and the Docker daemon. Provide a comma-separated list of host names, domain
 names, or wildcard host names that do not use the defined proxy. By default,
 this list is augmented with the list of all defined {product-title} host names.
 
+The host names that do not use the defined proxy include:
+
```

Comment:
Missing domain suffix from hostname.
E.g: when hostname is master.internal.example.com, the domain suffix - ".internal.example.com" should be listed in `openshift_no_proxy`